components/nds-simple-pagination/nds-simple-pagination.d.ts

```diff
@@ -0,0 +1,17 @@
+declare module "@nice-digital/nds-simple-pagination" {
+	import * as React from "react";
+
+	interface PageLinkProps {
+		destination?: string;
+		elementType?: React.ElementType;
+	}
+
+	export interface SimplePaginationProps {
+		currentPage: number;
+		totalPages?: number;
+		nextPageLink: PageLinkProps | null;
```

Comment:
Then you don't need to do: `nextPageLink={{ destination: "#next" }}` you can just do `nextPageLink="#next"`.

Also, does it make sense to be `nextPageLink?: PageLinkProps | null;` (undefined too) so that you don't have to explicilty pass null

Reply:
This is a nice idea but would be out of step with the other components that take links in this shape - do you think that's a concern?

Reply:
Fair enough, let's leave it then - can always add it in the future